native/source/mixpanel/detail/worker.cpp

```diff
@@ -190,6 +196,50 @@ namespace mixpanel
             }
         }
 
+        int Worker::parse_www_retry_after(nanowww::Response& response)
```

Comment:
I believe this will fail later since the methods on `nanowww::Response` are not marked `const`